stations.csv

```diff
@@ -5079,205 +5079,205 @@ id;name;slug;uic;uic8_sncf;latitude;longitude;parent_station_id;is_city;country;
+6672;Cordoue;cordoue;7150500;;;;;f;ES;f;Europe/Madrid;t;ESODB;;f;;f;7100038;f;;f;;f;;f;;f;;f;50500;t;f;;;;Cordoba;;;;;;;;;;;;;;
+6673;Port Aventura;port-aventura;7165411;;;;;f;ES;f;Europe/Madrid;t;ESPAV;;f;;f;7100229;f;;f;;f;;f;;f;;f;65411;t;f;;;;Port Aventura;;;;;;;;;;;;;;
+6674;Plasencia Ciudad;plasencia-ciudad;7130002;;;;;f;ES;f;Europe/Madrid;t;ESPLC;;f;;f;7100157;f;;f;;f;;f;;f;;f;30002;t;f;;;;Plasencia;;;;;;;;;;;;;;
+6675;Pamplona;pamplona;7180100;;41.378911;2.140367;;f;ES;f;Europe/Madrid;t;ESPNA;;f;;f;7100080;f;;f;;f;;f;;f;;f;80100;t;f;;;;Pamplona/Iruña;;;;;;;;;;;;;;
 6676;Port-Bou;port-bou;;;;;;t;ES;f;Europe/Madrid;f;ESPOB;;f;;f;;f;;f;;f;;f;;f;;f;;;f;;;;;;;;;;;;;;;;;;
-6677;Tarragona;tarragona;7171500;;;;;f;ES;f;Europe/Madrid;f;ESQGN;;f;;f;;f;;f;;f;;f;;f;;f;;;f;;;;;;;;;;;;;;;;;;
+6677;Tarragona;tarragona;7171500;;;;;f;ES;f;Europe/Madrid;t;ESQGN;;f;;f;7100062;f;;f;;f;;f;;f;;f;71500;t;f;;;;Tarragona;;;;;;;;;;;;;;
 6678;Ribas de Freser;ribas-de-freser;7177303;;;;;f;ES;f;Europe/Madrid;f;ESRDF;;f;;f;;f;;f;;f;;f;;f;;f;;;f;;;;;;;;;;;;;;;;;;
-6679;Redondela de Galicia;redondela-de-galicia;7122300;;;;;f;ES;f;Europe/Madrid;f;ESRDG;;f;;f;;f;;f;;f;;f;;f;;f;;;f;;;;;;;;;;;;;;;;;;
+6679;Redondela de Galicia;redondela-de-galicia;7122300;;;;;f;ES;f;Europe/Madrid;t;ESRDG;;f;;f;7100380;f;;f;;f;;f;;f;;f;22300;t;f;;;;Redondela;;;;;;;;;;;;;;
 6680;Las Rosas;las-rosas;;;;;;t;ES;f;Europe/Madrid;f;ESRSS;;f;;f;;f;;f;;f;;f;;f;;f;;;f;;;;;;;;;;;;;;;;;;
-6681;San Fernando de Cádiz;san-fernando-de-cadiz;7151402;;;;;f;ES;f;Europe/Madrid;f;ESSFD;;f;;f;;f;;f;;f;;f;;f;;f;;;f;;;;;;;;;;;;;;;;;;
-6682;Salamanca;salamanca;7130100;;41.133002;0.859998;;f;ES;f;Europe/Madrid;f;ESSLM;;f;;f;7171304;f;;f;;f;;f;;f;;f;;;f;;;;;;;;;;;;;;;;;;
+6681;San Fernando de Cádiz;san-fernando-de-cadiz;7151402;;;;;f;ES;f;Europe/Madrid;t;ESSFD;;f;;f;;f;;f;;f;;f;;f;;f;51402;t;f;;;;San Fernando De Cadiz;;;;;;;;;;;;;;
+6682;Salamanca;salamanca;7130100;;41.133002;0.859998;;f;ES;f;Europe/Madrid;t;ESSLM;;f;;f;7100027;f;;f;;f;;f;;f;;f;30100;t;f;;;;Salamanca;;;;;;;;;;;;;;
 6683;Sevilla;sevilla;;;;;;t;ES;f;Europe/Madrid;f;ESSVQ;;f;;f;;f;;f;;f;;f;;f;;f;;;f;;;;;;;;;;;;;;;;;;
-6684;Camp Tarragona;camp-tarragona;7104104;;41.19206141706671;1.272541880607605;;f;ES;f;Europe/Madrid;t;ESTAA;;t;;f;;f;;f;;f;;f;;f;;f;;;f;;;;;;;;;;;;;;;;;;
-6685;Torremolinos;torremolinos;7154509;;;;;f;ES;f;Europe/Madrid;f;ESTRM;;f;;f;;f;;f;;f;;f;;f;;f;;;f;;;;;;;;;;;;;;;;;;
-6686;Tuy;tuy;7122401;;;;;f;ES;f;Europe/Madrid;f;ESTUY;;f;;f;;f;;f;;f;;f;;f;;f;;;f;;;;;;;;;;;;;;;;;;
+6684;Camp Tarragona;camp-tarragona;7104104;;41.19206141706671;1.272541880607605;;f;ES;f;Europe/Madrid;t;ESTAA;;t;;f;;f;;f;;f;;f;;f;;f;04104;t;f;;;;Camp Tarragona;;;;;;;;;;;;;;
+6685;Torremolinos;torremolinos;7154509;;;;;f;ES;f;Europe/Madrid;t;ESTRM;;f;;f;;f;;f;;f;;f;;f;;f;54509;t;f;;;;Torremolinos;;;;;;;;;;;;;;
+6686;Tuy;tuy;7122401;;;;;f;ES;f;Europe/Madrid;t;ESTUY;;f;;f;;f;;f;;f;;f;;f;;f;22401;t;f;;;;Tui /Tuy;;;;;;;;;;;;;;
```

Comment:
In this kind of cases, I'd put just Tui in the info_es field. For now, "Tuy (Tui / Tuy)" will be displayed in the apps, there is a repetition.